lib/active_fedora/attributes.rb

```diff
@@ -126,6 +126,11 @@ def system_attributes
         ['has_model', 'create_date', 'modified_date']
       end
 
+      # Attributes inherited from Fedora in ActiveFedora::FedoraAttributes
+      def inheritable_attributes
```

Comment:
I got it from this method:
https://github.com/projecthydra/active_fedora/blob/master/lib/active_fedora/fedora_attributes.rb#L25
But I'm not attached to the name. I was hoping for a way to pull the names of the accessors from the FedoraAttributes module, but didn't see a clear way to do this, so I just opted to repeating the terms here.